pepper-apis/studybuilder-cli/studies/basil/study-events.conf

```diff
@@ -155,6 +155,22 @@
       "delaySeconds": null,
       "dispatchToHousekeeping": false,
       "order": 9
+    },
+
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": ${id.act.prequal},
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "ACTIVITY_INSTANCE_CREATION",
+        "activityCode": ${id.act.boolean},
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": false,
+      "order": 8
```

Comment:
The order seems to be copy pasted. I suggest either moving the trigger to the end of the list or re-enumerate all following triggers.